src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_transferIn.kt

```diff
@@ -742,7 +740,7 @@ class OffendersResourceTransferImpTest : ResourceTest() {
                 BedAssignmentHistory::getAssignmentEndDate,
               )
               .containsExactly(
-                tuple("ADM", bookingInTime.toLocalDate(), LocalDate.now()),
+                tuple("ADM", bookingInTime.toLocalDate(), transferOutDateTime.toLocalDate()),
                 tuple("19", transferOutDateTime.toLocalDate(), null),
```

Comment:
this tests seems wrong, and I think it relied on data by the builder *not* being set correctly, since the above indicated the bed would have been released when they transferred out